configure.ac

```diff
@@ -802,22 +802,41 @@ AC_ARG_VAR([VCC_CC], [C compiler command line for VCL code])
 if test "$ac_cv_env_VCC_CC_set" = "set"; then
 	VCC_CC="$ac_cv_env_VCC_CC_value"
 else
+	case $target in
+	*-*-darwin*)
+		VCC_CC="cc $OCFLAGS"
+		;;
```

Comment:
Why not `$CC`?

Reply:
> Why not `$CC`?

no idea. I just preserved what was there before:

https://github.com/varnishcache/varnish-cache/pull/3799/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810L817

Reply:
Someone with a vested interest in macos should probably look at this, but for this pull request it's out of scope.